java-util/src/main/java/io/druid/java/util/common/parsers/ParserUtils.java

```diff
@@ -94,4 +94,25 @@ public static String stripQuotes(String input)
     }
     return input;
   }
+
+  public static String replace(String text, String searchString, String  replacement)
```

Comment:
String.replace() and String.replaceAll() all String's method, they can't deal with null, this method encapsulation complexity, I will add the test case.